metadata/terms.go

```diff
@@ -17,6 +17,10 @@
 
 package metadata
 
+// CurrentTermsVersion is the version which has to be agreed
+// with when confirming TOS agreements.
+const CurrentTermsVersion = "0.0.27"
```

Comment:
Should this be changed at https://github.com/mysteriumnetwork/terms?
I thought that this file is generated and should not be edited.

Reply:
Maybe someone else could have input on this? As far as I can see, this isn't generated, tho I could be wrong.

Reply:
AFAIK this is generated from the terms and should not be changed manually.

Reply:
As we don't embed terms library to node, but rather show link to https://github.com/mysteriumnetwork/terms
Decided to embed just terms version instead, to known which terms he/she was reading when accepting terms.